posenet/README.md

```diff
@@ -57,13 +57,16 @@ All keypoints are indexed by part id.  The parts and their ids are:
 Single pose estimation is the simpler and faster of the two algorithms. Its ideal use case is for when there is only one person in the image. The disadvantage is that if there are multiple persons in an image, keypoints from both persons will likely be estimated as being part of the same single pose—meaning, for example, that person #1’s left arm and person #2’s right knee might be conflated by the algorithm as belonging to the same pose.
 
 ```javascript
-const pose = await poseNet.estimateSinglePose(image, outputStride);
+const pose = await poseNet.estimateSinglePose(image, inputSize, reverse, outputStride);
 ```
 
 #### Inputs
 
-* **image** - a 3d tensor of the image data to predict the poses for.
-* **outputStride** - the desired stride for the outputs when feeding the image through the model.  Must be 32, 16, 8.   The higher the number, the faster the performance but slower the accuracy, and visa versa.
+* **image** - ImageData|HTMLImageElement|HTMLCanvasElement|HTMLVideoElement
+   The input image to feed through the network.
+* **inputSize** - The size to scale the image to before feeding through the network. Defaults to 513.  Must have a value which when 1 is subtracted from it, is divisible by the output stride. Sample acceptable values are 29, 161, 193, 257, 289, 321, 353, 385, 417, 449, 481, 513. Set this number lower to scale down the image and increase the speed when feeding through the network.
```

Comment:
I'm wondering if we should change this to a scale factor, and then we can automatically adjust the scale factor to be one of these values.